content/recipes/applied-examples/approach-cdisc.md

```diff
@@ -0,0 +1,167 @@
+
+The recipe will cover the following elements:
+
+> * Provide a general overview of the challenges of mapping a non-conforming data dictionary to CDISC.
+
+> * Illustrate the mapping process using the above projects.
+
+> * Describe the ETL processes necessary to convert the data to CDISC-SDTM based on the mapped data dictionary.
+
+---
+
+
+## Graphical Overview
+
+````{dropdown}
+:open:
+```{figure} approach-cdisc.md-figure0.mmd.png
+---
+name: approach-cdisc-figure0
+alt: Recipe Steps
+---
+Recipe Steps
+```
+````
+
+
+---
+
+
+## Requirements
+
+* Technical requirements:
+   * Knowledge of relational databases is an advantage
+   * Fluency in a scripting language such as bash, Python or R
+* Knowledge requirement:
+   * A thorough understanding of CDISC standards, in particular CDISC-SDTM, is essential
+   
+
+---
+
+<!--## FAIRification Objectives, Inputs and Outputs
+
+
+| Actions.Objectives.Tasks  | Input | Output  |
+| :------------- | :------------- | :------------- |
+| [text annotation](http://edamontology.org/operation_3778)  | local data dictionary  | [annotated text](http://edamontology.org/data_3779)  |-->
+
+
+## Table of Data Standards
+
+
+| Data Formats  | Terminologies | Models  |
+| :------------- | :------------- | :------------- |
+|  NA |  CDISC terminology | CDISC-SDTM  |
+
+---
+
+## General challenges of mapping to CDISC-SDTM
+
+[CDISC-SDTM](https://www.cdisc.org/standards/foundational/sdtm) is a standard model and framework for organising, annotating and formatting data from clinical trials. Regulatory agencies such as the US FDA require clinical trial results to be submitted in this format. While SDMT is a very complex framework with a high learning curve, its ubiquity in the field of clinical trials makes it a useful interoperability tool for comparing and combining datasets. The SDTM Implementation Guide ([SDTMIG](https://www.cdisc.org/standards/foundational/sdtmig)) provides expanded guidance on implementing SDTM for specific use cases or "domains". A general introduction to the SDTMIG is available [on the CDISC website](https://www.cdisc.org/standards/foundational/sdtmig/primer).
+
+Given the complexity of CDISC standards, any project team intending conversion of their datasets to SDTM at any point in the project life cycle should aim to align with the standard as early on in the process as possible. Data dictionaries and data collection instruments should where possible be aligned to the relevant CDISC standards to facilitate data conversions later on. In particular, CDISC standards provide procedures and guidelines for encoding project-specific data that might not fit into the existing domains.
+
+Commercial solutions are available to address all of these use cases but these are outside the scope of this recipe. We will focus on outlining how to perform the mapping process manually, without bespoke tooling.
```

Comment:
````suggestion
```{note}
Commercial solutions are available to address all of these use cases but these are outside the scope of this recipe. 

We will focus on outlining how to perform the mapping process manually, without bespoke tooling.
```
````